src/AsmResolver/AsmResolver.csproj

```diff
@@ -33,4 +33,8 @@
         <PackageReference Include="MonoMod.Backports" Version="1.1.0" />
     </ItemGroup>
 
+    <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0'">
+        <PackageReference Include="System.Memory" Version="4.5.5" />
+    </ItemGroup>
```

Comment:
I am not convinced we should do this, especially because we can easily avoid it (see other comments on unmanaged data source). This will pull in 3 additional packages for .NET standard 2.0 targets, which is not great for .NET Framework 4.x targets:

- [System.Buffers](https://www.nuget.org/packages/System.Buffers/) (>= 4.5.1)
- [System.Numerics.Vectors](https://www.nuget.org/packages/System.Numerics.Vectors/) (>= 4.4.0)
- [System.Runtime.CompilerServices.Unsafe](https://www.nuget.org/packages/System.Runtime.CompilerServices.Unsafe/) (>= 4.5.3)